sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/polling/implementation/PollingConstants.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.util.polling.implementation;
+
+/**
+ * INTERNAL CLASS.
+ *
+ * The constants use for polling related operations.
+ */
+public final class PollingConstants {
+    /**
+     * The header used for location based polling or sending the final result URL.
+     */
+    public static final String LOCATION = "Location";
+    /**
+     * The header used for location based polling or sending the final result URL.
+     */
+    public static final String LOCATION_LOWER_CASE = "location";
+    /**
+     * The key for storing the request URL in the polling context.
+     */
+    public static final String REQUEST_URL = "requestURL";
+    /**
+     * The key for storing the request HTTP method in the polling context.
+     */
+    public static final String HTTP_METHOD = "httpMethod";
+    /**
+     * The key for storing the final resource location in the polling context.
+     */
+    public static final String RESOURCE_LOCATION = "resourceLocation";
+    /**
+     * The key for storing the initial resource data in the polling context.
+     */
+    public static final String INITIAL_RESOURCE_RESPONSE_BODY = "initialResourceResponseBody";
```

Comment:
Added key for initial resource response body.

Reply:
This key can be added to azure-core's `PollingConstants` itself. Since it's in the implementation package of core, removing or changing this constant will not be considered breaking change.